### ISSUE TITLE
About model training speed

### ISSUE THREAD

**Issue by XiaChaoStudio** [bekou/multihead_joint_entity_relation_extraction#8]
Why is the model training speed very slow？ 1M training set, 100k test set, training test takes about a day, and uses 4 GPUs at the same time.

**Comment by bekou** [comment]
I have never tried to use a GPU with this model. FYI, using a Intel(R) Xeon(R) CPU E5-2620 v4 @ 2.10GHz cpu, takes around 40 seconds per epoch for the CoNLL04 dataset (training set 0.5 mb).
